.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -217,11 +217,57 @@ jobs:
       if: always()
       run: |
           if [[ "${{ job.status }}" == "failure" || "${{ job.status }}" == "cancelled" ]]; then echo "status=${{ job.status }}" >> $GITHUB_OUTPUT; fi
+
+  update_grafana_dashboard:
+    name: update_grafana_dashboard
+    needs: [ initialize_nightly, workload ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Generate grafana dashboard.json using grafonnet container
+        run: |
+          sudo podman run --rm --name run_grafonnet -v "$GITHUB_WORKSPACE"/benchmark_runner/grafana/perf:/app --privileged quay.io/ebattat/run_grafonnet:latest
+
+      - name: Check for changes in dashboard.json
+        id: check_changes
+        run: |
+          # Backup dashboard.json before updating the Grafana dashboard to avoid changes in file indentation
+          cp -p "$GITHUB_WORKSPACE"/benchmark_runner/grafana/perf/dashboard.json "$GITHUB_WORKSPACE"/benchmark_runner/grafana/perf/dashboard_backup.json
```

Comment:
Can't you put this line in the update Grafana dashboard immediately before the `python "$GITHUB...` line, before the file could be modified?

Reply:
Fixed it